src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Steps/ConfirmIdentitySteps.cs

```diff
@@ -14,46 +17,85 @@ namespace SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests.Steps
     public class ConfirmIdentitySteps : StepsBase
     {
         private readonly TestContext _context;
+        private Guid _registrationId = Guid.NewGuid();
 
         public ConfirmIdentitySteps(TestContext context) : base(context)
         {
             _context = context;
         }
 
-        [Given(@"the apprentice has logged in")]
-        public void GivenTheApprenticeHasLoggedIn()
+        [Given("the apprentice has not logged in")]
+        public void GivenTheApprenticeHasNotLoggedIn()
         {
         }
 
-        [Given(@"the apprentice has not verified his identity")]
-        public void GivenTheApprenticeHasNotVerifiedHisIdentity()
+        [Given("the apprentice has logged in")]
+        public void GivenTheApprenticeHasLoggedIn()
         {
+            TestAuthenticationHandler.AddUser(_registrationId);
+            _context.Web.Client.DefaultRequestHeaders.Authorization =
+                new AuthenticationHeaderValue(_registrationId.ToString());
         }
 
-        [When(@"first accessing the commitment statement website")]
-        public async Task WhenFirstAccessingTheCommitmentStatementWebsite()
+        [Given("the apprentice has not verified their identity")]
+        public void GivenTheApprenticeHasNotVerifiedTheirIdentity()
         {
-            var request = new HttpRequestMessage(
-                HttpMethod.Get,
-                "account");
-           await _context.Web.Get("account");
+            _context.OuterApi.MockServer.Given(
+                Request.Create()
+                    .UsingGet()
+                    .WithUrl(_context.OuterApi.UrlPath($"registrations/{_registrationId}"))
+                    )
+                .RespondWith(Response.Create()
+                    .WithStatusCode(200)
+                    .WithBodyAsJson(new
```

Comment:
nice I didn't know this existed! Does that also mean the

.WithHeader("Content-Type", "application/json")

is automatically added?